internal/broker/bind_create.go

```diff
@@ -45,8 +48,11 @@ type Credentials struct {
 	Kubeconfig string `json:"kubeconfig"`
 }
 
-func NewBind(cfg BindingConfig, instanceStorage storage.Instances, log logrus.FieldLogger, clientProvider broker.ClientProvider, kubeconfigProvider broker.KubeconfigProvider, gardenerClient client.Client) *BindEndpoint {
-	return &BindEndpoint{config: cfg, instancesStorage: instanceStorage, log: log.WithField("service", "BindEndpoint"),
+func NewBind(cfg BindingConfig, instanceStorage storage.Instances, bindingsStorage storage.Bindings, log logrus.FieldLogger, clientProvider broker.ClientProvider, kubeconfigProvider broker.KubeconfigProvider, gardenerClient client.Client) *BindEndpoint {
+	return &BindEndpoint{config: cfg,
+		instancesStorage:             instanceStorage,
+		bindingsStorage:              bindingsStorage,
+		log:                          log.WithField("service", "BindEndpoint"),
```

Comment:
Is there any reason we use `logrus` not `slog`?

Reply:
Logrus is used in our broker api implementation but here we could change it. However, it is out of scope of this task.

Reply:
The idea was to introduce slog everywhere (unless this is not feasible due to other requirements).
To minimize work on migration of entire KEB code base to slog.

Reply:
But at this stage let's leave it as it is.